src/sinks/amqp/config.rs

```diff
@@ -8,12 +8,40 @@ use crate::{
 };
 use codecs::TextSerializerConfig;
 use futures::FutureExt;
+use lapin::{types::ShortString, BasicProperties};
 use std::sync::Arc;
 use vector_config::configurable_component;
 use vector_core::config::AcknowledgementsConfig;
 
 use super::sink::AmqpSink;
 
+/// AMQP Properties options.
```

Comment:
Should properties be capitalized or no? The configurable title below does not capitalize it. Beyond what makes sense from the spec of the protocol I think we should be consistent.

Reply:
Changed for "AMQP properties configuration", as the name of the struct suggest it's configuration instead of options.